objectivec/GPBCodedInputStream.h

```diff
@@ -35,6 +35,34 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+CF_EXTERN_C_BEGIN
+
+/// GPBCodedInputStream exception name.
+extern NSString *const GPBCodedInputStreamException;
+
+/// NSError domain used for GPBCodedInputStream errors.
+extern NSString *const GPBCodedInputStreamErrorDomain;
```

Comment:
This seems a little out of place, as the class only raises exceptions.  I'd expect message to pick them off via the exception name/prefix and map them to errors at that point?